### ISSUE TITLE
Add support for flexible environment variable settings

### ISSUE THREAD

**Issue by yorinasub17** [gruntwork-io/helm-kubernetes-services#59]
e.g being able to specify:

```
    spec:
      containers:
      - image: foobar:latest
        name: go-test-monitor-container
        imagePullPolicy: Always
        env:
          - name: DD_AGENT_HOST
            valueFrom:
              fieldRef:
                fieldPath: status.hostIP
          - name: DD_ENTITY_ID
            valueFrom:
              fieldRef:
                fieldPath: metadata.uid 
```

**Comment by yorinasub17** [comment]
Suggested implementation is to expose an input value called `additional_container_env` which can be used to pass in raw list items that are rendered for the `env` map.

**Comment by jjneely** [comment]
Implementation: https://github.com/gruntwork-io/helm-kubernetes-services/issues

**Comment by yorinasub17** [comment]
This is now released as `v0.1.1`
